aptos-move/move-examples/token_objects/sources/aptos_token.move

```diff
@@ -30,6 +29,33 @@ module token_objects::aptos_token {
     const ETOKEN_NOT_BURNABLE: u64 = 4;
     /// Attempted to mutate a property map that is not mutable
     const EPROPERTIES_NOT_MUTABLE: u64 = 5;
+    // The collection does not exist
+    const ECOLLECTION_DOES_NOT_EXIST: u64 = 1;
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    /// Storage state for managing the no-code Collection.
+    struct AptosCollection has key {
+        /// Used to mutate collection fields
+        mutator_ref: Option<collection::MutatorRef>,
+        /// Used to mutate royalties
+        royalty_mutator_ref: Option<royalty::MutatorRef>,
+        /// Determines if the creator can mutate the collection's description
+        mutable_description: bool,
```

Comment:
nit: Shouldn't this be description_mutable or description_mutability? mutable_description sounds like description (but mutable)

Reply:
the description is mutable or it has a mutable description. :shrug:

Reply:
I prefer `mutable_description` as mutable describes description and then from an alphabetical perspective it keeps them all together. mutable -> adjective